### PR TITLE
Add desktop version

### PR DESCRIPTION
**Pull request by Rafiduzzaman**
##This pull request includes updates to the media query for the desktop version and resolves padding and margin issues.

Changes Made:
- Updated the media query to ensure proper styling for desktop screens.
- Fixed padding and margin issues that were causing layout inconsistencies.

Purpose:
The purpose of these changes is to improve the appearance and responsiveness of the website on desktop devices. By refining the media query and resolving padding and margin issues, we aim to provide a better user experience and ensure consistent design across different screen sizes.

Testing:
- Tested the changes on various desktop devices and verified that the media query triggers correctly.
- Verified that the padding and margin issues are resolved and the layout appears as expected.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -603,16 +618,48 @@ body {
...
     background-repeat: no-repeat;
     background-origin: content-box;
-    background-position: 100% -10%;
-    background-size: 35%;
+    background-position: 100% -0.5%;
+    background-attachment: scroll;
+    background-size: 500px;
     display: flex;
     flex-direction: column;
   }
```
Comment (stanleySimeon) [inline style.css:679]: Greeting!

You're doing a great work, but according to the Figma design, there are a few things you'll need to change to go to the next steps.

- Kindly, I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> ## The Figma design
>
>Screenshot 2023-06-15 at 11 31 24 PM
>
> ## Your Design
>
>Screenshot 2023-06-15 at 11 04 20 PM

File: style.css [hunk 2]
```diff
@@ -658,105 +715,443 @@ body {
...
+  }
+
+  .container {
+    margin: 0 2vh;
+    padding: 0% 0%;
+    display: flex;
+    width: 150%;
+    flex-direction: column;
+    align-items: flex-start;
+  }
```
Comment (stanleySimeon) [inline style.css]: As mentioned above, kindly I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> ## The Figma design
>
>Screenshot 2023-06-15 at 11 36 09 PM
>Screenshot 2023-06-15 at 11 36 33 PM
>
> ## Your Design
>
>Screenshot 2023-06-15 at 11 08 33 PM
>Screenshot 2023-06-15 at 11 10 25 PM

File: style.css [hunk 3]
```diff
@@ -658,105 +715,443 @@ body {
...
+    background-origin: content-box;
+    background-position: 100%;
+    background-attachment: scroll;
+    background-size: 500px;
+    display: flex;
+    flex-direction: column;
+    background-color: #262626;
+    margin-bottom: 0%;
+    padding-bottom: 0%;
+  }
```
Comment (stanleySimeon) [inline style.css:1028]: As mentioned above, kindly I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> ## The Figma design
>
>Screenshot 2023-06-15 at 11 38 15 PM
>
> ## Your Design
>
>Screenshot 2023-06-15 at 11 13 22 PM
>Screenshot 2023-06-15 at 11 16 16 PM

File: style.css [hunk 4]
```diff
@@ -658,105 +715,443 @@ body {
...
+
+  #headerrr {
+    text-align: left;
+  }
+
+  #form1 {
+    display: grid;
+    width: 50%;
+    margin: 10vh 0 0 10vh;
+  }
```
Comment (stanleySimeon) [inline style.css]: As mentioned above, kindly I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> ## The Figma design
>
>Screenshot 2023-06-15 at 11 41 54 PM
>
> ## Your Design
>
>Screenshot 2023-06-15 at 11 20 49 PM

**Review by stanleySimeon — CHANGES_REQUESTED** [review]

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @Rdzaman20000, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Meaningfull commit messages
- Good PR description

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shella12 — APPROVED** [review]
Hi @Rdzaman20000, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great design implementation ✔️ 
- Implemented all required changes 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**